scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -466,6 +466,19 @@ class Router(formatOps: FormatOps) {
           Split(NewlineT(isDouble = tok.hasBlankLine), 0)
         )
 
+      case tok @ FormatToken(_: T.RightParen, _: T.KwDef | _: T.Comment, _)
+          if leftOwner.is[Defn.ExtensionGroup] =>
+        val expireToken = tok.right match {
+          case _: T.KwDef => leftOwner.tokens.last
+          case _ => nextNonComment(next(tok)).right
```

Comment:
why right? looks inconsistent with the KwDef handling.

Reply:
`right` will be the `def` token. It doesn't work with `left`, which will be still a comment.

Reply:
so for comment, you will return `KwDef`. but in case of `KwDef`, you are not returning the `KwDef` token, you are returning the last token of the entire `ExtensionGroup`.

wouldn't `leftOwner.tokens.last` be sufficient for both cases? or `lastToken(leftOwner)` (which stops before trailing comments/whitespace)? i am not clear why there's asymmetry here.

Reply:
I was overthinking it, `leftOwner.tokens.last` works for both cases, thanks!